### PR TITLE
Update Copernicus to v6.4 and add missing Copernicus journal

### PR DESCRIPTION
**Pull request by RLumSK**
Another update of the Copernicus template (now version 6.4) and I've added a so far missing journal ("Web Ecology") to `copernicus_article()`

- [x] Update NEWS.

### CODE REVIEW

**Comment by RLumSK** [comment]
@nuest Sorry, I forgot to ping you for this PR.

**Comment by RLumSK** [comment]
@cderv The failed CI checks do not seem to be related to my PR.

**Comment by cderv** [comment]
Yes I agree. It seems to be from the Glossa format. 

our test suites runs every format for now. It would be interesting to only test the modified format for PR 🤔

**Comment by RLumSK** [comment]
@cderv Thanks for the prompt response. For what it is worth, I have tested the modified version, only the Copernicus template, manually on my computer (macOS 11.6.1, pandoc 2.16.1, R 4.1.1). No problem here, but perhaps this is not a replacement for the full tests you run.

File: R/copernicus_article.R [hunk 1]
```diff
@@ -77,18 +77,17 @@ copernicus_journals <- list(
...
   "Archives Animal Breeding" = "aab",
-  "ASTRA Proceedings" = "ap",
   "Atmospheric Chemistry and Physics" = "acp",
   "Atmospheric Measurement Techniques" = "amt",
   "Biogeosciences" = "bg",
   "Climate of the Past" = "cp",
   "DEUQUA Special Publications" = "deuquasp",
   "Drinking Water Engineering and Science" = "dwes",
-  "European Journal of Mineralogy" = "ejm",
   "Earth Surface Dynamics" = "esurf",
```
Comment (cderv) [inline R/copernicus_article.R:86]: Do we really need to deleted them ? 

This will probably break some code if one was using for example 
````r
rticles::copernicus_journal_abbreviations(journal_name = '"ASTRA Proceedings')
````

Can't we just keep them ? 

possibly in a separate list if we want to differentiate new and old journal name ?

File: NEWS.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # rticles 0.22
 
-
+- Update Copernicus Publications template to version 6.4 from 2021-08-16
```
Comment (cderv) [inline NEWS.md:3]:
```suggestion
- Update Copernicus Publications template to version 6.4 from 2021-08-16 (thanks, @RLumSK, #446).
```

**Review by cderv — APPROVED** [review]
Looks good overall apart from a comment I left above. 

After you answer and possibly change, we are good to merge

File: R/copernicus_article.R (hunk 1 above)
Comment (cderv) [reply R/copernicus_article.R:86]: > If the journal is not known, the PDF still builds (I've have tested it with rticles + plain LaTeX). 

This is if the function `rticles::copernicus_journal_abbreviations` was not used to derive the journal abbreviation like in the template YAML header, right ? 

> If a journal is not published anymore by Copernicus, so it is unlikely that new articles are written for such journals.

Good point. 

But I like your warning solution to possibly warn there is something not expected. 

> I intended to keep the list precisely as Copernicus has it (easier to compare, and we do not drag old stuff with us)

What about keeping a list of old journal in a separate object and use this list to warn that this is an old journal ? 

What do you think ?

Comment (RLumSK) [reply R/copernicus_article.R:86]: > This is if the function rticles::copernicus_journal_abbreviations was not used to derive the journal abbreviation like in the template YAML header, right ?

No, I used the function, e.g., `r rticles::copernicus_journal_abbreviations(journal_name = 'gaga')` returns 
just something that is empty. More precise, the corresponding code returns `NULL`, which is then included in the template. 

> What about keeping a list of old journal in a separate object and use this list to warn that this is an old journal ?

There is just one problem. I would only know about the list now, and I have no idea what Copernicus did in the past, and I am not sure whether there is a record. Hence, I would prefer to keep that very general. What do you think?

Comment (cderv) [reply R/copernicus_article.R:86]: > More precise, the corresponding code returns NULL, which is then included in the template.

And what happens in that case when no journal is passed ? Just want to be sure to understand everything. THanks

Comment (RLumSK) [reply R/copernicus_article.R:86]: ## Three scenarios: 

1. `rticles::copernicus_journal_abbreviations(journal_name = "")`: All journal abbreviations are passed onto LaTeX; looks odd but the PDF builds.
2. `rticles::copernicus_journal_abbreviations()`: Same as 1.
3. `rticles::copernicus_journal_abbreviations("non-existing journal")`: returns `NULL` and in the LaTeX nothing. 

### Scenario 1 and 2: Tex output

```tex
\documentclass[adgeo, ars, asr, ascmo, angeo, aab, acp, amt, bg, cp,
deuquasp, dwes, esurf, esd, essd, egqsj, ejm, fr, gchron, gh, gc, gi,
gmd, hgss, hess, jbji, jm, jsss, mr, ms, nhess, npg, os, pb, piahs,
sand, sd, soil, se, tc, we, wcd, we, wes, manuscript]{copernicus}
```

### Scenario 3: Tex output

```tex
\documentclass[, manuscript]{copernicus}
```

Bottom line, it is rather robust.

Comment (cderv) [reply R/copernicus_article.R:86]: Ok let's leave it that way then. Thanks for the explanation and the PR

**Comment by RLumSK** [comment]
@cderv Thanks for your support and the critical PR review! You raised an important point I had overlooked, at least now we have tested that and I can keep an eye on it, in case it causes some problems in the future.
